### PR TITLE
v1.5.2

### PR DESCRIPTION
**Pull request by mobileoverlord**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.5.2
+
+* Enhancements
+  * Added `NERVES_DATA_DIR` environment variable for specifying the location
+    for global artifacts and downloads.
+  * Configure `NERVES_DATA_DIR` to `XDG_DATA_HOME` if defined.
+  * Add `nerves` config for `source_date_epoc` to enable reproducible builds.
```
Comment (fhunleth) [inline CHANGELOG.md]: * Nerves tooling now supports setting the `SOURCE_DATE_EPOCH` environment variable for reproducible builds during compilation via `:source_date_epoch` in your application config. This removes timestamp differences between builds. See [reproducible-builds.org](https://reproducible-builds.org/) for more information.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  * Allow `erlinit.config` options to be overridden from Mix config.
+    The options from the system are merged with the options specified in the
+    config for `:nerves, :erlinit`.
```
Comment (fhunleth) [inline CHANGELOG.md]: * `erlinit.config` options can be overridden using the application config now. For example, in your `config.exs` you can now add:

```
config :nerves, :erlinit,
  ctty: "ttyAMA0"
```

I think we need to update the docs for this feature. I couldn't find any documentation except for the source code. Here's what we currently say: https://github.com/nerves-project/nerves/blob/553de3a139402ed47b7ba307eb1379ab017f0225/docs/Advanced%20Configuration.md#overwriting-files-in-the-root-filesystem

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.5.2
+
+* Enhancements
+  * Added `NERVES_DATA_DIR` environment variable for specifying the location
```
Comment (fhunleth) [inline CHANGELOG.md]: I think this one is just:

  * Support `XDG_DATA_HOME`. If `XDG_DATA_HOME` is set, Nerves will now store its data under that directory.

This feels like the least important enhancement, so my vote would be to list it last. The `erlinit` one feels like the most important one or at least the one that's caused the most annoyance.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+    for global artifacts and downloads.
+  * Configure `NERVES_DATA_DIR` to `XDG_DATA_HOME` if defined.
+  * Add `nerves` config for `source_date_epoc` to enable reproducible builds.
+  * Allow `erlinit.config` options to be overridden from Mix config.
+    The options from the system are merged with the options specified in the
+    config for `:nerves, :erlinit`.
+
+* Bug fixes
+  * Improved support for identifying Windows Subsystem for Linux in mix task
```
Comment (fhunleth) [inline CHANGELOG.md]: This seems like an improvement rather than a bug fix.
  * Windows Subsystem for Linux improvements

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,23 @@
...
+    preflight checks.
+  * Improved error messages when missing host packages.
```
Comment (fhunleth) [inline CHANGELOG.md]: Skip this one.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,23 @@
...
+  * Do not require sudo on `mix burn` if already privileged.
+  * Keep start and start_clean boot scripts when making a release.
```
Comment (fhunleth) [inline CHANGELOG.md]: * Keep all boot scripts. Previously, extraneous boot scripts from the OTP release process were removed. Keeping them makes it possible to start Erlang slave nodes and support use cases where triggers at device boot time launch different scripts.

File: CHANGELOG.md (hunk 2 above)
Comment (mobileoverlord) [reply CHANGELOG.md]: I added a blurb about it here https://github.com/nerves-project/nerves/blob/553de3a139402ed47b7ba307eb1379ab017f0225/docs/Advanced%20Configuration.md#overriding-erlinitconfig-from-mix-config

Maybe we need to move it above the section mentioned?

**Pull request merged**